### DISCUSSION TITLE
使用unplugin-vue-components自动化按需引入组件的，怎么汉化啊？

### DISCUSSION THREAD

**Discussion by elarbor** [element-plus/element-plus#19297] [Q&A]
组件默认是英文的
![image](https://github.com/user-attachments/assets/25c692a5-8b79-4ef0-90fa-3701caec3bfa)

**Comment by Xmppp** [comment]
![微信图片编辑_20241217161401](https://github.com/user-attachments/assets/651fce93-3cce-4bd4-8aab-1dc802b53edb)

**Reply by elarbor** [reply]
不是全局引入的element-plus。而是通过unplugin-vue-components自动化按需引入组件的

**Reply by WangJincheng4869** [reply]
> 不是全局引入的element-plus。而是通过unplugin-vue-components自动化按需引入组件的

![图片](https://github.com/user-attachments/assets/9756874c-0326-475b-895f-edb0594dedc5)

**Reply by elarbor** [reply]
[嗯嗯](https://element-plus.org/zh-CN/component/config-provider.html)
![image](https://github.com/user-attachments/assets/0bb09f38-dfec-4d90-9763-7be4688a666a)

好了
